.github/workflows/codecov.yaml

```diff
@@ -110,7 +110,7 @@ jobs:
 
   codecov-results:
     name: "Publish Coverage Results"
-    needs: codecov
+    needs: [codecov, bsim-test]
```

Comment:
That is not how `workflow_call` works. Please see https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow